apps/web/pages/api/auth/signup.ts

```diff
@@ -116,12 +117,18 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         });
       }
 
-      const membership = await prisma.membership.update({
+      const membership = await prisma.membership.upsert({
```

Comment:
membership doesn't exist when inviting by "copy link".

Reply:
Is it worth adding a comment in the code itself?